[2024-03-04T13:09:07.855+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.group_database.create_table_task manual__2024-03-04T13:09:03.321236+00:00 [queued]>
[2024-03-04T13:09:07.864+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.group_database.create_table_task manual__2024-03-04T13:09:03.321236+00:00 [queued]>
[2024-03-04T13:09:07.864+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T13:09:07.865+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-03-04T13:09:07.865+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T13:09:07.876+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): group_database.create_table_task> on 2024-03-04 13:09:03.321236+00:00
[2024-03-04T13:09:07.881+0000] {standard_task_runner.py:55} INFO - Started process 148 to run task
[2024-03-04T13:09:07.883+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'gera_dados_turbina', 'group_database.create_table_task', 'manual__2024-03-04T13:09:03.321236+00:00', '--job-id', '3234', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp1mn4e8tp']
[2024-03-04T13:09:07.885+0000] {standard_task_runner.py:83} INFO - Job 3234: Subtask group_database.create_table_task
[2024-03-04T13:09:07.936+0000] {task_command.py:388} INFO - Running <TaskInstance: gera_dados_turbina.group_database.create_table_task manual__2024-03-04T13:09:03.321236+00:00 [running]> on host 9c7e4f3ae948
[2024-03-04T13:09:07.995+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=adrianolima095@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gera_dados_turbina
AIRFLOW_CTX_TASK_ID=group_database.create_table_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-04T13:09:03.321236+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-04T13:09:03.321236+00:00
[2024-03-04T13:09:07.996+0000] {sql.py:254} INFO - Executing: create table if not exists sensors 
                                        (  idtemp            varchar
                                         , powerfactor       varchar
                                         , hydraulicpressure varchar
                                         , temperature       varchar
                                         , timestamp         varchar
                                         );
                                    
[2024-03-04T13:09:08.002+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2024-03-04T13:09:08.211+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2024-03-04T13:09:08.214+0000] {sql.py:375} INFO - Running statement: create table if not exists sensors 
                                        (  idtemp            varchar
                                         , powerfactor       varchar
                                         , hydraulicpressure varchar
                                         , temperature       varchar
                                         , timestamp         varchar
                                         );
                                    , parameters: None
[2024-03-04T13:09:08.236+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=gera_dados_turbina, task_id=group_database.create_table_task, execution_date=20240304T130903, start_date=20240304T130907, end_date=20240304T130908
[2024-03-04T13:09:08.255+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-03-04T13:09:08.274+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
